Cap3 fab5 23.1

Biossalus opened the pull request:
File Structure & Naming conventions are ready. 
Have a look and let us know if we are missing any Classes/Interfaces.

Biossalus committed:
initial spring package commit

Biossalus committed:
Files Structure & Naming Conventions

1Fungi99 reviewed (changes requested):
Need to add "node_modules" to git ignore

AbigailSwigert commented on manager/src/main/java/inventory/manager/Services/IProductController.java:
```diff
@@ -0,0 +1,4 @@
+package inventory.manager.Services;
+
+public interface IProductController {
```
Is this supposed to be IProductService? There is already a ProductController and it is not an interface.

AbigailSwigert reviewed (changes requested):
I believe the IProductController file has the wrong name. Other than that, looks great. This had made me realize that we need to add an admin table to our database schema, I think.

Biossalus committed:
Added dependancies to the pom.xml
refactored Admin & Customer classes/Interfaces into a combined class 'Account'
Abstracted Entities into a separate package.
'Model' package is now for 'info' classes.
Added packages for the following:
Config
Entity
Utils
Validator
-------
Fixes:
-Added *.node_modules to .gitignore
-Corrected name of IProductController -> IProductService

Biossalus commented:
Updated branch to reflect the concerns addressed:
Added *.node_modules to .the .gitignore
Fixed the name on IProductController -> IProductService

Added more packages to the structure with their corresponding classes/interfaces
Added Dependencies to the pom.xml
Refactored Admin & Customer classes/interfaces into a single 'Account' class/interface

We will still need that Admin table in the DB.

PrettyScript reviewed (commented):
I noticed that there is a "PaginationResult Class" I'm not sure if we would need that class, from what we learned I was under the impression that we would just extend "PagingAndSortingRepository" from the corresponding repository. We could always add/delete classes, this doesn't have to be set & stone.

Dylan-Fellows reviewed (approved):
Everything looks good. Great work!
